src/turnstyl.ts

```diff
@@ -136,7 +132,7 @@ class Turnstyl {
         if (!this.deepCompareKeys(producerSchema, dbPayload)) {
           // add to log file when theres error
           logger.error(
-            `The database payload and producer event have a field (key) mistmatch
+            `❌ The database payload and producer event have a field (key) mistmatch
```

Comment:
```suggestion
            `The database payload and producer event have a field (key) mistmatch
```
Again, I think this may throw error with an emoji due to changes within our test platform